index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="style.css">
+        <title>Portfolio | Edwin Gichuhi</title>
+    </head>
+
+    <body>
+        <nav class="toolbar">
+            <a href="#">John Doe</a>
+            <img src="/images/Icon-Menu.png" alt="menu">
+        </nav>
+        <section class="headline">
+            <h1>Hey There. I'm Alex</h1>
+            <h2>I’am a Software Developer</h2>
+            <h3>I can help you build a product , feature or website Look through some of my work and experience! If you
+                like what you
+                see and have a project you need coded,
+                don’t hestiate to contact me.</h3>
+            <section class="socials">
+                <img src="/images/Icon-GitHub.png" alt="Github">
+                <img src="/images/Icon-Linkedin.png" alt="Github">
+                <img src="/images/Vector.png" alt="Github">
+                <img src="/images/Icon-Twitter.png" alt="Github">
+                <img src="/images/Icon-Meta.png" alt="Github">
```

Comment:
:red_circle: As you can see from the screenshot below :arrow_down: none of your images is displaying. I think there is something wrong with your image paths (**hamburger menu, social icons**). Please check your image paths. I can only see the alternative texts rather than the images.

 :bulb: **Note: When adding a relative path. Please Make sure your start with a dot` .` to target your current directory your path should be similar to `src="./images/Portfolio 1.png" `Notice the dot at the beginning**
### Your Design
![studentdesign28](https://user-images.githubusercontent.com/26736582/198685237-48b7999a-891f-4f90-ab0b-7170eda5c279.png)

### Figma Design

![figma design28](https://user-images.githubusercontent.com/26736582/198685294-8d0494c2-dcf5-44b9-aa08-5c52520adaa0.png)

Reply:
Thanks for this. Fixed.